app/components/avo/fields/common/single_file_viewer_component.html.erb

```diff
@@ -1,3 +1,4 @@
+<% return unless persistent_record? %>
```

Comment:
We can use the built-in `render?` function on `single_file_viewer_component.rb`

https://viewcomponent.org/guide/conditional_rendering.html

Reply:
Didn't know that exists, just applied!